static-assets/components/cstudio-admin/base.js

```diff
@@ -21,14 +21,20 @@
 
   CStudioAdminConsole = {
     toolContainerEls: [],
+    isEmbedded: false,
```

Comment:
You could directly check here if the embedded class is present, right? No need to delay.